tests/tests/unit-tests/routers/OrganizationRouterTest.js

```diff
@@ -123,6 +123,13 @@ describe ('OrganizationRouter', function () {
             // always return done() to continue the test chain
             return done();
           });
+
+          it ('a new admin should have been created', function (done) {
+            request (blueprint.app.server.app)
+              .get ('/v1/admin/users/' + userId)
+              .set ('Authorization', 'bearer ' + adminAccessToken)
+              .expect (200);
```

Comment:
Make sure to add a call to the done callback.  You can do this by changing .expect (200); to .expect (200, done);